build.gradle

```diff
@@ -1,30 +0,0 @@
-plugins {
```

Comment:
Why you removed root project's build.gradle file?

Reply:
Because every scenario is it's own Gradle project. It's not a sub-module. 
So we dont' need any gradle files at project root level.